src/actions/guides/database/testing.cr

```diff
@@ -7,8 +7,20 @@ class Guides::Database::Testing < GuideAction
 
   def markdown
     <<-MD
+    ## Introduction
+
+    A Box is an object used to map a corresponding [model](#{Guides::Database::Models.path}) to your
```

Comment:
I think  this first sentence may be more confusing than helpful until you see  an example. Thoughts on removing it and starting with the second one? "A Box is a class that  allows  youth easily create test data"